drivers/gpio/gpio_pca953x.c

```diff
@@ -467,7 +479,9 @@ static const struct gpio_driver_api api_table = {
 			.port_pin_mask = GPIO_PORT_PIN_MASK_FROM_DT_INST(n),	\
 		},								\
 		.interrupt_enabled = DT_INST_NODE_HAS_PROP(n, nint_gpios),	\
-		.gpio_int = GPIO_DT_SPEC_INST_GET_OR(n, nint_gpios, {0}),	\
+		.gpio_int = GPIO_DT_SPEC_INST_GET_OR(n, nint_gpios, {0}),  \
```

Comment:
alignment of `\` is off, resulting in unnecessary line change

Reply:
Thank you for your comment, has been updated!